pkg/ocm/billing.go

```diff
@@ -38,18 +40,73 @@ func (c *Client) GetBillingAccounts() ([]string, error) {
 	for _, item := range quotaCostResponse.Items().Slice() {
 		for _, account := range item.CloudAccounts() {
 			if account.CloudProviderID() == "aws" && !helper.Contains(billingAccounts, account.CloudAccountID()) {
-				billingAccounts = append(billingAccounts, account.CloudAccountID())
+				contracts, hasContract := account.GetContracts()
+				var contractString string
+				if hasContract {
+					dimensions := contracts[0].Dimensions()
+					for _, dimension := range dimensions {
+						if GetDimensionValue(dimension, "four_vcpu_hour") > 0 || GetDimensionValue(dimension, "control_plane") > 0 {
+							contractString = " [Contract enabled]"
+						}
+					}
+				}
+				billingAccounts = append(billingAccounts, account.CloudAccountID()+contractString)
 			}
 		}
 	}
 
 	if len(billingAccounts) == 0 {
-		return billingAccounts, errors.New("no billing accounts found")
+		return billingAccounts, errors.New("No billing account associated. " +
+			"Go to https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html" +
+			"to learn how to associate your billing account." +
+			" You must associate a billing account to continue.")
 	}
 
 	return billingAccounts, nil
 }
 
+func (c *Client) GetBillingAccountContracts(cloudAccoutID string) ([]*v1.Contract, error) {
```

Comment:
I don't think  we need a new function for this. We can probably refactor/modify `GetBillingAccounts`.

Reply:
k will work on that

Reply:
Removed GetBillingAccounts.